src/Composer/Satis/Builder/ArchiveBuilder.php

```diff
@@ -53,15 +59,55 @@ public function dump(array $packages)
         $archiveManager->setOverwriteFiles(false);
 
         shuffle($packages);
+
+        $progressBar = null;
+        $hasStarted = false;
+        $verbosity = $this->output->getVerbosity();
+        $isStats = $this->input->getOption('stats') && OutputInterface::VERBOSITY_NORMAL == $verbosity;
+
+        if ($isStats) {
+            $packageCount = 0;
+
+            foreach ($packages as $package) {
+                if (!$helper->isSkippable($package)) {
+                    ++$packageCount;
+                }
+            }
+
+            $this->composer->setDownloadManager($this->composer->getDownloadManager());
```

Comment:
None, it's a mistake, maybe a wrong copy/paste. Didn't see it, thanks.

Should I create a new PR ?